server/env.js

```diff
@@ -16,6 +19,19 @@ if (!process.env.NODE_CONFIG_ENV) {
 }
 
 dotenv.config();
+
+// Load extra env file on demand
+// `npm run dev staging` / `npm run dev production`
+if (process.env.EXTRA_ENV || process.env.OC_ENV === 'development') {
+  const extraEnv = process.env.EXTRA_ENV || last(process.argv);
+  if (['local', 'staging', 'production'].includes(extraEnv)) {
```

Comment:
Having a limited set is helpful to make this safe, though being able to define custom envs could also be nice:

```bash
$ ls
.env.staging
.env.local
.env.local-prod-copy
...

$ npm run dev local-prod-copy
```

Reply:
@Betree Do you want me to relax? Should not hurt?

Reply:
I think so, especially since the file needs to be prefixed with `.env.`. It seems safe